tests/test_exapp.py

```diff
@@ -61,8 +61,9 @@ def load_arguments(self, command):
 
 class TestMyScenarios(ScenarioTest):
 
-    def __init__(self, method_name):
-        super(TestMyScenarios, self).__init__(mycli, method_name)
+    def setUp(self):
+        super().setUp()
+        self.cli = mycli
```

Comment:
Initializing `self.cli` in `setUp` adheres to the "do heavy work in `setUp`" principle: https://github.com/Azure/azure-cli/pull/28849